public/401.html

```diff
@@ -0,0 +1,88 @@
+
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta property="og:image" content="/assets/opengraph-image.png">
+    </head>
+
+    <body class="controller-sessions">
+
+        <div id="skiplink-container">
+            <div>
+                <a href="#content" class="skiplink">Skip to main content</a>
+            </div>
+        </div>
+
+        <header role="banner" id="global-header" class="with-proposition">
+            <div class="header-wrapper">
+                <div class="header-global">
+                    <div class="header-logo">
+                        <a href="/" title="Return to home page" id="logo" class="content">
+                            <img src="/assets/gov.uk_logotype_crown_invert_trans.png" width="36" height="32" alt=""> Track a query
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </header>
+
+        <div id="global-header-bar"></div>
+
+        <div class="grid-row"><div class="column-full">
+            <main id='page-container' role='main'>
+                <div id="wrapper" class="group live">
+                    <section id="content" role="main">
+                        <header class="page-header group">
+                            <h1 class="heading-xlarge">Sorry, you are not authorised to view this page</h1>
+                        </header>
+                        <p>Please contact <a href="mailto:correspondence@digital.justice.gov.uk?subject=400">correspondence@digital.justice.gov.uk</a> with the details on how you got here to report this issue.</p>
+                    </section>
+                </div>
+
+            </main>
+
+        </div></div>
+
+        <footer class="group js-footer" id="footer" role="contentinfo">
+            <div class="footer-wrapper">
+                <div class="footer-meta">
+                    <div class="footer-meta-inner">
+                        <div class="open-government-licence">
+                            <p class="logo"><a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" rel="license">Open Government Licence</a></p>
+                            <p>All content is available under the <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" rel="license">Open Government Licence v3.0</a>, except where otherwise stated</p>
+                        </div>
+                    </div>
+
+                    <div class="copyright">
+                        <a href="http://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/copyright-and-re-use/crown-copyright/">© Crown copyright</a>
+                    </div>
+                </div>
+            </div>
+        </footer>
+
+        <div id="global-app-error" class="app-error hidden"></div>
+
+        <script>
+            (function (i, s, o, g, r, a, m) {
```

Comment:
could we extract this out to a module so it is not declared twice on separate pages, or is there a reason for doing it this way?